geniza/corpus/templatetags/corpus_extras.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def alphabetize(value):
+    """Lowercases, then alphabetizes, a list of strings"""
+    if isinstance(value, list) and all(isinstance(s, str) for s in value):
+        return sorted([s.lower() for s in value])
+    else:
+        raise TypeError("Argument must be a list of strings")
```

Comment:
I'm a little surprised you needed a custom template tag for this. I'm not sure that lower-casing the tags is the desired behavior, either... oh, but of course we're displaying all caps so case doesn't matter.

Ok, makes sense and I'm good with this approach.  

For consistency, we should apply to the document detail page as well — but if it doesn't work as-is we can hold off (I think we have a queryset of taggit tags instead of a list of strings, so may need some adjustment).

Reply:
Yeah, weirdly Django's only builtin filter/template tag for sorting is `sortdict`, which requires a `dict` input rather than a `list`. And exactly on the lowercasing—I was so confused for a minute about why they weren't displaying alphabetically, then I realized some had an uppercase first letter and some didn't!

Indeed it's not a list of strings in the document detail page, so probably good to hold off for #306 or thereabouts—went ahead and added it into the PR desc there so I don't forget.